CAM Platform/api-providers.md

```diff
@@ -144,16 +145,14 @@ ElasticBox-Release: 4.0
 }
 ```
 
-**VSphere request parameters**
-
+#### VSphere request parameters
```

Comment:
Please change any occurrence of _VSphere_ to _vSphere_ and of _VCloud_ to _vCloud_ (in order to be aligned with the labels we use in CAM UI).